editor/scene_tree_dock.cpp

```diff
@@ -4764,6 +4771,7 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	scene_tree->connect("files_dropped", callable_mp(this, &SceneTreeDock::_files_dropped));
 	scene_tree->connect("script_dropped", callable_mp(this, &SceneTreeDock::_script_dropped));
 	scene_tree->connect("nodes_dragged", callable_mp(this, &SceneTreeDock::_nodes_drag_begin));
+	scene_tree->get_scene_tree()->get_vscroll_bar()->connect(SNAME("value_changed"), callable_mp(this, &SceneTreeDock::_reset_hovering_timer_from_scroll));
```

Comment:
```suggestion
	scene_tree->get_scene_tree()->get_vscroll_bar()->connect(SNAME("value_changed"), callable_mp(this, &SceneTreeDock::_reset_hovering_timer).unbind(1));
```

Then there would be no need for a new method, that one line would be enough

Reply:
Forgot about unbind. Just pushed the change